components/blocks/DirsToCardSections.vue

```diff
@@ -35,11 +35,6 @@ export default {
     CardGroup,
   },
   filters: {
-    humanize(str) {
-      const cleanStr = str.replace(/-/g, ' ');
```

Comment:
urlize is already in the methods section, delete the filter

Reply:
on line 9, change that usage to method instead of filter.

I don't see humanize used (though I could have missed it) - delete?